Sources/Linq2DynamoDb.DataContext/EntityWrapper.cs

```diff
@@ -14,6 +16,31 @@ internal class EntityWrapper : IEntityWrapper
         private readonly IEntityKeyGetter _keyGetter;
         private Document _doc, _newDoc;
 
+        private PropertyInfo _entityVersionNumberProperty;
+        private bool _hasResolvedEntityVersionNumberProperty;
+
+        /// <summary>
+        /// Gets PropertyInfo for the entity's property that has [DynamoDBVersion] 
+        /// attribute or returns null if there is none.
+        /// </summary>
+        private PropertyInfo EntityVersionNumberProperty {
+            get {
+                if (!_hasResolvedEntityVersionNumberProperty) {
```

Comment:
not sure about the project's braces style, but looks like you have mixed `foo {` and `foo\n{` in this file's changes.